mlflow/ml-package-versions.yml

```diff
@@ -212,14 +212,14 @@ fastai-1.x:
 
   models:
     minimum: "1.0.60"
-    maximum: "1.0.61"
+    maximum: "1.0.61" # IGNORE_UPDATE: 2.x is not supported yet
     requirements: ["scikit-learn"]
     run: |
       pytest tests/fastai/test_fastai_model_export.py --large
 
   autologging:
     minimum: "1.0.60"
-    maximum: "1.0.61"
+    maximum: "1.0.61" # IGNORE_UPDATE: 2.x is not supported yet
```

Comment:
Why not we add the IGNORE_UPDATE config by a field like:

autologging:
    minimum: "1.0.60"
    maximum: "1.0.61"
    ignore_update: "2.x"

the comment way looks trigger when parsing.

Reply:
I agree. Adding a new field makes more sense. Let me fix.